src/util/Util.ts

```diff
@@ -205,7 +205,13 @@ export function parseData(data: unknown, contentType: string): any {
                     return data;
                 }
                 case "json": {
-                    return JSON.parse(data as string);
+                    return JSON.parse(data);
+                }
+                case "integer": {
+                    return parseInt(data);
+                }
+                case "float": {
+                    return parseFloat(data);
                 }
```

Comment:
I don't understand why you put data in strings and then parse it, except for maybe the base64. Can't you transfer objects, arrays, and numbers as they are?

Reply:
Strings with a content type kind of keep things manageable. Pyodide has proper conversion methods, but other programming environments might not.  The current data type in BackendEvent for "data" is string, so I stuck to that for now instead of adding numbers to the mix.